runtime/oti/vmconstantpool.xml

```diff
@@ -31,6 +31,8 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<classref name="java/lang/InstantiationError"/>
 	<classref name="java/lang/UnsatisfiedLinkError"/>
 	<classref name="java/lang/InternalError"/>
+	<classref name="java/lang/IdentityObject"/>
```

Comment:
Would this cause an issue if J9VM_OPT_VALHALLA_VALUE_TYPES is not defined?

Reply:
yes, you can add flags see `	<classref name="java/lang/invoke/FilterArgumentsHandle" flags="opt_methodHandle"/>
`